data/json/furniture_and_terrain/terrain-railroads.json

```diff
@@ -562,5 +562,73 @@
       "items": [ { "item": "pipe", "count": [ 2, 3 ] }, { "item": "scrap", "count": [ 0, 2 ] } ]
     },
     "flags": [ "TRANSPARENT", "NOITEM" ]
+  },
+  {
+    "id": "t_railroad_track_small",
+    "type": "terrain",
+    "name": "small railroad track",
+    "description": "Like a railroad track, only smaller. You could probably run a small vehicle on this.",
+    "symbol": "X",
+    "color": [ "cyan" ],
+    "move_cost": 3,
+    "looks_like": "t_railroad_track",
```

Comment:
Ok for now it looks like a single rail, but with an updated tileset it'll look like two rails, works for me.